README.md

```diff
@@ -51,7 +54,23 @@ An example project to show one use case or workflow for creating "variants".  Th
 	* Build all bundles
 	* Remove the extra files/groups that were created.
   * After building with "Pack Variations", you can enter play mode using the standard "Packed Play Mode" script.
- 
+   
+#### *Advanced/Sync Addressables*
+Synchronous Addressables!  What a crazy thing.  This sample shows how you could set up your project to utilize Addressables, but in a (nearly) completely synchronous way.
+Why don't we put these sync methods in Addressables itself?  The best way to understand that is to look at SyncAddressables/SyncAddressables.cs and search for "throw".  The code is very specific about how it needs to be used, and will cause pain for the caller if not used in the right way at the right time.  That being said, if you want to create a game built on sync interfaces, you can copy this code, and run with it.  If you are using it, all the existing async methods would still work, so you are capable of doing a mix & match in your game, if you are willing to accept the constraints when diong things sync. Note that the group schema is what associates a given asset group with either the sync providers or the regular ones.  So you could not mix & match within a group.
+*Packed Mode and Fast Mode only*.  We have not implemented a sample script for Virtual mode.  Note that for packed mode, you actually use the standard packed script.  For packed mode, it's the bundle building that needs a custom script, not the 
```

Comment:
the very end of the last sentence trails off "...not the" and then it just ends.  If the list below is meant to be things that don't need the custom script, add a ":".  Or if it's just not showing up right on the diff for this PR let me know